base/setup/usetup/lang/es-ES.h

```diff
@@ -136,61 +146,61 @@ static MUI_ENTRY esESIntroPageEntries[] =
     {
         4,
         3,
-        " ReactOS " KERNEL_VERSION_STR " Setup ",
+        " Instalaci�n de ReactOS " KERNEL_VERSION_STR " ",
         TEXT_STYLE_UNDERLINE
     },
     {
         6,
         8,
-        "ReactOS Version Status",
+        "Acerca del estado de ReactOS",
         TEXT_STYLE_HIGHLIGHT
     },
     {
         6,
         11,
-        "ReactOS is in Alpha stage, meaning it is not feature-complete",
+        "ReactOS est� en estado Alpha, es decir, no es un sistema completo",
```

Comment:
Double space?

Reply:
The "á" character, (a with acute) in CP437 to the char code 160, as we can see here: https://freebasic.net/wiki/wikka.php?wakka=CptAscii

in windows 1252 and so on, the char code 160 corresponds to a non-break blank-space, or &nbsp; html entity.

Reply:
Duh, of course. Thanks.

Reply:
Maybe it could be interesting to do what @Fraizeraust did for italian translations, namely using `\x` escape sequences.